src/url/bench/email.results.txt

```diff
@@ -0,0 +1,33 @@
+✓ src/url/bench/email.bench.js > url/email passing all tests 9054ms
+     name                                               hz     min     max    mean     p75     p99    p995    p999     rme  samples
+   · `EmailWithURL.canParse` (current)        2,179,868.71  0.0003  1.6480  0.0005  0.0005  0.0007  0.0008  0.0024  ±0.83%  1089935
+   · `EmailClassWithRegExp.canParse`         12,644,349.22  0.0000  0.1205  0.0001  0.0001  0.0001  0.0001  0.0003  ±0.07%  6322175   fastest
+   · `emailWithRegExp`                       12,591,750.06  0.0000  0.0951  0.0001  0.0001  0.0001  0.0001  0.0003  ±0.09%  6295876
+   · `validator/isEmail`                        687,041.51  0.0013  1.0840  0.0015  0.0015  0.0018  0.0018  0.0023  ±0.57%   343521
+   · yup create schema and `isValidSync`        583,325.45  0.0015  1.0215  0.0017  0.0017  0.0023  0.0025  0.0039  ±0.65%   291663   slowest
+   · yup precalculated schema `isValidSync`   2,119,568.48  0.0004  0.8205  0.0005  0.0005  0.0009  0.0010  0.0014  ±0.35%  1059785
+
+ ✓ src/url/bench/email.bench.js > url/email not passing all tests 6586ms
```

Comment:
All libraries in this category (joi, Valibot, Zod) are only failing on edge cases.

Reply:
Elaborating in https://github.com/frontacles/frontacles/pull/35#issuecomment-2708188805.